scrapinghub/client.py

```diff
@@ -203,6 +195,17 @@ def _extract_spider_id(self, params):
             raise ValueError('Please use same spider id')
         return spiderid
 
+    def update_tags(self, add=None, remove=None, spidername=None):
+        spidername = spidername or (self.spider.name if self.spider else None)
+        if not spidername:
+            raise ValueError('Please provide spidername')
```

Comment:
^ worth discussing with @michalmo 

if changes are hard or undesirable on the dash side - we can stick with ValueError for now, just move it to a reusable method.

renaming `spidername` to `spider` in the client could be confusing, I'd expect `spider` to be an instance of `Spider`
```

```